reactor-core/src/main/java/reactor/core/publisher/Sinks.java

```diff
@@ -112,92 +132,73 @@ private Sinks() { }
 	 */
 	@SuppressWarnings("deprecation")
 	public static <T> Sink<T> unicast() {
-		return new ProcessorSink<>(UnicastProcessor.create());
+		UnicastProcessor<T> processor = UnicastProcessor.create();
+		return SinksHelper.toSink(processor, processor);
 	}
 
 	/**
 	 * A {@link Sink} that works like a conceptual Promise: it can be completed
 	 * with or without a value at any time, but only once. This completion is replayed to late subscribers.
-	 * Calling {@link Sink#next(Object)} is enough and does not require a following {@link Sink#complete()}.
+	 * Calling {@link Sink#emitNext(Object)} is enough and does not require a following {@link Sink#emitComplete()}.
```

Comment:
add ` (which is implicit).` at the end

Reply:
also add that as a result the `Sinks` closely follows the `Mono` contract and is better suited to be viewed as `toMono()` (hinting at 0-overhead conversion to mono vs overhead conversion to flux)